tests/js/spec/views/performance/landing/widgets/widgetContainer.spec.jsx

```diff
@@ -920,6 +920,95 @@ describe('Performance > Widgets > WidgetContainer', function () {
       );
     });
 
+    it('P95 Duration - with null values', async function () {
```

Comment:
We could remove some duplicate assertions in this test to keep it light and only assert on things that are different from the normal P95 Duration test.